### DISCUSSION TITLE
Email Size Limit

### DISCUSSION THREAD

**Discussion by trikotret** [simple-login/app#664] [Q&A] [answered]
Hi,
In FAQ section it says email size limit is 25mb. Yesterday someone tried to email me a 19.70mb file but it got rejected due to size. So I had that person just send it to my gmail directly and I received it fine. The file was indeed 19.70mb. I decided to test myself so I sent the same file from my  proton mail thru SL to get forwarded to my gmail and it also got rejected. I got this error "message size 26665980 exceeds size limit 25600000 of server mx1.simplelogin.co" The file is only 19.70mb. Any idea why SL is seeing the file size 26mb? What's the real email limit size attachment so I can avoid this in the future?

**Comment by nguyenkims** [comment, marked as answer]
The only cause I can think of is the file is somehow bigger when it leaves ProtonMail. This can be possible due to the email encoding which might increase the email size. We have increased our limit to a little bit more than 25 MB to take into account this case.

**Reply by trikotret** [reply]
HI
I did another test with a 24mb file. It got rejected. **"**message size 33640509 exceeds size limit 30000000 of = server mx1.simplelogin.co[94.237.111.15]"**** I tried sending the same file thru gmail and PM and both service providers received file. When I tried to use SL, it got rejected. For some reason these email providers and making the file size larger by 10mb. I see you increased the email limit from 25mb to 30mb. Maybe increase it to 35mb and  hopefully that'll work for a real 25mb file.

**Reply by nguyenkims** [reply]
@trikotret we have increased the limit again and should now be enough for covering this case :).

**Comment by 0xspade** [comment]
Hi @nguyenkims,

Can we also do this in self-hosted instance?

**Reply by nguyenkims** [reply]
This is rather a Postfix limit. It can be set using the `message_size_limit` in Postfix.
